component/tests/vshn.yml

```diff
@@ -152,7 +152,7 @@ parameters:
                   appuio.io/node-class: "plus"
               note: "Will be scheduled on APPUiO Cloud plus nodes"
         minio:
-          enabled: false
+          enabled: true
```

Comment:
Can you reset this to false? For local dev a minio instance will already be provisioned via kindev, so we can save a bit of resource by not running minio twice.